### PR TITLE
Files

### PR DESCRIPTION
**Pull request by asiajanicka**

### CODE REVIEW

File: src/main/java/org/jjm/TripBooking.java [hunk 1]
```diff
@@ -19,16 +16,28 @@
...
 
-        TravelAgency travelAgency = new TravelAgency();
-        logger.info("Travel agency initialized with possible destinations");
+        TravelAgency travelAgency = null;
+        try {
+            travelAgency = new TravelAgency();
+            logger.info("Travel agency initialized with possible destinations");
+        } catch (InvalidDataException e) {
+            logger.error("Travel agency couldn't be created due to wrong init destination data", e);
+            throw new RuntimeException(e);
```
Comment (vboltrukanis) [inline src/main/java/org/jjm/TripBooking.java]: do not use runtime exceptions like that

File: src/main/java/org/jjm/TripBooking.java [hunk 2]
```diff
@@ -286,5 +295,11 @@ public static void main(String[] args) {
         logger.debug(String.format("Customized trip - participant added: %s ", participantTom.getPerson()));
 
         myTrip.printSummary();
+
+        try {
+            Utils.writeSessionStatisticsToFile("src/test/resources/tempStatistics.txt");
```
Comment (vboltrukanis) [inline src/main/java/org/jjm/TripBooking.java]: lets try to configure this path from config file

File: src/main/java/org/jjm/bookings/TransportBooking.java [hunk 3]
```diff
@@ -9,15 +9,18 @@
 import java.math.BigDecimal;
 
 public abstract class TransportBooking<T> implements ICost {
+    private static int counter = 0;
```
Comment (vboltrukanis) [inline src/main/java/org/jjm/bookings/TransportBooking.java:12]: what do you use this counter for?

**Review by vboltrukanis — CHANGES_REQUESTED** [review]
do not use RuntimeException and small refactoring required

File: src/main/java/org/jjm/TripBooking.java (hunk 1 above)
Comment (asiajanicka) [reply src/main/java/org/jjm/TripBooking.java]: changed

File: src/main/java/org/jjm/TripBooking.java (hunk 2 above)
Comment (asiajanicka) [reply src/main/java/org/jjm/TripBooking.java]: I added reading paths from config file for this one and for  the file with data to initialize the hotel too.

File: src/main/java/org/jjm/bookings/TransportBooking.java (hunk 3 above)
Comment (asiajanicka) [reply src/main/java/org/jjm/bookings/TransportBooking.java:12]: I use it only to print total number of transport bookings to the file with statistics

**Review by vboltrukanis — APPROVED** [review]

**Pull request merged**